docroot/themes/contrib/civictheme/civictheme_library/components/00-base/_variables.components.scss

```diff
@@ -244,6 +244,34 @@ $civictheme-chip-dark-active-background-color: civictheme-color('primary', 4) !d
 $civictheme-chip-dark-active-border-color: civictheme-color('neutral', 1) !default;
 $civictheme-chip-dark-active-outline-color: civictheme-color('outline') !default;
 
+//
```

Comment:
Moved SCSS variables to the right “group”

They are part of atoms and were added at end.